### DISCUSSION TITLE
Comparisons between this and box86/box64?

### DISCUSSION THREAD

**Discussion by expenses** [FEX-Emu/FEX#4113] [Q&A]
Hi, there are currently a bunch of applications that use box86/64 for running windows games on Android. See https://github.com/brunodev85/winlator/releases for one example. I'm curious as to whether box86/64 could be replaced with fex for this purpose, and whether there would be any benefits in terms of performance or otherwise?

**Comment by Sonicadvance1** [comment]
FEX-Emu doesn't support Android. I'm not sure what that project's integration is like to know if FEX-Emu can be slotted in as a replacement.
I haven't ever done any comparisons against Box and FEX-Emu to know if there would be any significant performance differences, but we've been steadily improving our CPU emulation performance over the years.

**Reply by EthanPeng-2048** [reply]
A Chinese android app called Game fusion actually used fexemu to run x64 on arm

**Comment by Torinde** [comment]
Related:
- https://github.com/ptitSeb/box64/issues/2020
